modules/op-enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,57 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/creating-applications-with-cicd-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects_{context}"]
+= Enabling monitoring of event listeners for Triggers for user-defined projects
+
+As a cluster administrator, to gather event listener metrics for Triggers in a user-defined project and display them in the {product-title} web console, you can create a service monitor for each event listener. On receiving a HTTP request, event listeners for Triggers return three metrics -- `http_duration_seconds`, `event_count`, and `triggered_resources`.
+
+[discrete]
+.Prerequisites
```

Comment:
Not sure why make these discrete headings? I don't think it's required.

Reply:
To learn more, I've searched openshift-docs and looked at:

- https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#discrete-headings
- https://docs.asciidoctor.org/asciidoc/latest/blocks/discrete-headings/

There are many instances of these `[discrete]` attributes above block headings (e.g., `.Prerequisites`) in openshift-docs. However, they seem unnecessary because asciidoc already excludes block headings from TOCs and such. 
It seems that this attribute only affects section titles (e.g., `== This is a section title`) that you want to make more discrete.